### PR TITLE
reenable dynamic shadows

### PR DESCRIPTION
**Pull request by veprogames**
When setting sun and moon to visible = false, dynamic shadows won't display anymore. In combination with the high-res skyboxes, they increase graphical quality quite a bit. Setting sun and moon to an empty texture instead keeps dynamic shadows intact.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -56,8 +56,10 @@ skybox.set = function(player, number)
 				textures = textures,
 				clouds = true
 			})
-			player:set_sun({visible = false, sunrise_visible = false})
-			player:set_moon({visible = false})
+			-- making the sun invisible deactivates the dynamic shadows
+			-- making the sun visible with an invisible texture keeps shadows intact
+			player:set_sun({visible = true, sunrise_visible = false, texture = "skybox_invisible.png"})
+			player:set_moon({visible = true, texture = "skybox_invisible.png"})
```
Comment (SmallJoker) [inline init.lua]: `empty.png` is provided my Minetest. Might that be of interest for you?

Comment (veprogames) [reply init.lua]: I tried using empty.png as texture, but it wasn't found:

`ERROR[Main]: generateImage(): Could not load image "empty.png" while building texture; Creating a dummy image`

Comment (SmallJoker) [reply init.lua]: My bad. It's called `blank.png`.
https://github.com/minetest/minetest/blob/master/textures/base/pack/blank.png

Comment (veprogames) [reply init.lua]: using `blank.png` now, see latest commit

**Comment by SmallJoker** [comment]
Sorry, I totally forgot about this PR. LGTM. Merging.

**Pull request merged**